extensions/u301-url-shortener/package.json

```diff
@@ -11,32 +11,21 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "index",
-      "title": "Shorten URL",
+      "name": "shorten-clipboard",
```

Comment:
Can one of the two commands keep the same `name`. It's the unique ID of the command where ranking, aliases and hotkeys are saved